user_manual/groupware/contacts.rst

```diff
@@ -122,8 +122,63 @@ to create new address books, simply by specifying an address books name:
 .. figure:: ../images/contact_manageaddressbook.png
 	:alt: Add address book in the contacts settings
 
-The Contacts settings is also where you can share, export and delete address
-books. You will find the CardDAV URLs there.
+The Contacts settings is also where you can share, export and delete addressbooks. You will find the CardDAV URLs there.
 
 See :doc:`index` for more details about syncing your address books
 with iOS, macOS, Thunderbird and other CardDAV clients.
+
+
+Circles
+-------
+
+Informal collaboration takes place within organizations: an event to organize for a few weeks, a short ideation session between members from differents entities, workshops, a place to joke around and support team building, or simply in very organic organizations where formal structure is kept to a minimum.
+
+For all these reasons, Nextcloud supports Circles, a feature embedded in the Contacts app, where every user is able to create its own circle, a user-defined aggregate of accounts. Circles can be used later on to share files and folders, added to Talk conversations, like a regular group.
+
+.. figure:: ../images/circle.png
```

Comment:
This breaks building.
The circle image is missing

Reply:
@maximelehericy what image were you planning to add?

Reply:
https://github.com/nextcloud/documentation/pull/10979

Reply:
```suggestion
.. figure:: ./images/circle.png
```